packages/prisma/schema.prisma

```diff
@@ -171,6 +171,8 @@ model EventType {
   maxLeadThreshold                        Int?
   selectedCalendars                       SelectedCalendar[]
   allowReschedulingPastBookings           Boolean                   @default(false)
+  /// @zod.custom(imports.emailSchema)
+  customReplyToEmail                      String?
```

Comment:
Why not an actual relationship with VerifiedEmails?

Reply:
In this case, `VerfifiedEmails` are used to send a workflow to an email address, not necessarily replying to. We're using the user's primary email or a verified `SecondaryEmail`. 

![CleanShot 2025-04-23 at 09 30 03@2x](https://github.com/user-attachments/assets/eeb817bb-fac2-49a0-95d9-1b1017869663)